docs/content/operations/quickstart-migration.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: doc_page
+title: "Migrating Derby Metadata and Local Deep Storage"
```

Comment:
you can have a non-HA cluster with non local deep storage and derby metadata store

I think the metadata and deep storage migration should have separate docs

Reply:
Some food for thought:
when a user goes from single server to cluster, they should first read the deep storage migration doc

when a user goes from non-HA cluster to HA-cluster, they should read the metadata store migration doc

Reply:
Hm, there may be some redundancy in contents but I'll look into splitting

Reply:
Redundancy is fine as long as it is clear to the user what they should do.

Reply:
I made a separate page for the `export-metadata` tool and made separate pages for metadata/deep storage migration which reference the tool doc page